### ISSUE TITLE
How use in a React Component?

### ISSUE THREAD

**Issue by zjoin** [matthieua/WOW#234]
Is there an example of how to use on react?

**Comment by azat-io** [comment]
The same question

**Comment by renatorib** [comment]
If you are using webpack: http://stackoverflow.com/questions/35099440/wow-js-with-webpack-and-react

If not, thats no secret.

**Comment by jsantana90** [comment]
I'm getting errors and the answer from @renatorib doesn't seem to be my issue.

I do `import WOW from 'wowjs';`

```javascript
componentDidMount() {
  const wow = new WOW();
  wow.init();
}
```

but I get `TypeError: _wowjs2.default is not a constructor`

If I do `import { WOW } from 'wowjs';`

I get

```
MutationObserver is not supported by your browser.
WOW.js cannot detect dom mutations, please call .sync() after loading new content.
```

If I do 

```javascript
componentDidMount() {
  const wow = new WOW();
  wow.sync();
}
```

Notice the `wow.sync()`

I get `TypeError: Cannot read property 'nodeType' of undefined`

Stuck there :(

**Comment by Jaikant** [comment]
@johhansantana 

Your code worked for me, I just had to include the script into my html file 
<script src="//cdnjs.cloudflare.com/ajax/libs/wow/0.1.12/wow.min.js"></script>

if (typeof window !== 'undefined') {
 let WOW = require('wowjs');
}
  componentDidMount() {
    if (typeof window !== 'undefined') {
      const wow = new WOW();
      wow.init();
    }
  }

**Comment by adammoisa** [comment]
Supposedly you're supposed to double the WOW and write `new WOW.WOW().init()` but it still doesn't work. I can see in the rendered html that the animated class is indeed being added, but the animation doesn't run.

**Comment by QuadDepo** [comment]
@adammoisa you still need help to get it working?

**Comment by FrenchMajesty** [comment]
I have the same question.

**Comment by VayneHao** [comment]
@johhansantana  yeah. I have the same question .my code works when  I use the following.
 import WOW from 'wowjs/dist/wow.js';
componentDidMount() {
    const wow = new WOW();
    wow.init();
  }
But.I get WOW.js cannot detect dom mutations, please call .sync() after loading new content.
How do I slove the question?

**Comment by sallespro** [comment]
tried to browserify `dist/wow.js` and
```
  componentDidMount() {
    const WOW = require('../WOW/dist/wow.js')
    this.wow = new WOW.WOW()
    this.wow.init()
  }

```
but it logs
`e.WOW is not a constructor`

**Comment by phanthinh** [comment]
The same question

**Comment by FrenchMajesty** [comment]
Ok so what I did to get it working is to include the .js plugin in the <head> of my index.html:
`<script src="//cdnjs.cloudflare.com/ajax/libs/wow/1.1.2/wow.min.js" type="text/javascript"></script>`

And in my component: 
`
componentDidMount() {
        new WOW().init()
}
`

**Comment by monkeymonk** [comment]
Doing 
```
import {WOW} from 'wowjs/dist/wow';

new WOW().init();
```

Worked for me! ;-)

**Comment by macolo** [comment]
This should work out of the box with Webpack:

```
window.WOW = require('wowjs').WOW;
new WOW({
  mobile: false
}).init();
``` 

Why takes people from making their stuff compliant?

**Comment by 01Kuzma** [comment]
Hi!
In React.js I initiate this plugin with this code:
```
import WOW from 'wowjs';

class App extends React.Component {
  componentDidMount() {
    new WOW.WOW().init();
  }
...
```

But I cannot get rid of these warnings:
```
'MutationObserver is not supported by your browser.'
'WOW.js cannot detect dom mutations, please call .sync() after loading new content.'
```
They are showed in all browsers. I'm not adding something dynamically

How could it be fixed?

**Comment by vitorarantes** [comment]
@01Kuzma  

try this:
```
import WOW from 'wowjs';

componentDidMount() {
    new WOW.WOW({
        live: false
    }).init();
}
```
worked for me...

**Comment by phanthinh** [comment]
I have resolved it

**Comment by 01Kuzma** [comment]
@vitorarantes, nice! Works!

**Comment by goldy-suryan** [comment]
try this:
import { WOW } from "wowjs/dist/wow";

new WOW({
      live: false
    }).init();

**Comment by wiched** [comment]
This works for me:

```
import WOW from 'wowjs'

  componentDidMount() {
    if (typeof window !== 'undefined') {
      const wow = new WOW.WOW({
        live: false,
      })
      wow.init()
    }
  }
```

**Comment by VenkataJanapareddy** [comment]
```
import WOW from "wow.js/dist/wow.js";

componentDidMount() {
    new WOW().init();
  }

```

**Comment by momin-riyadh** [comment]
How can I init for functional components 
```javascript
 componentDidMount() {
        new WOW.WOW({
            live: false
        }).init();
    }
  ```

**Comment by MosiurRrahman** [comment]
who can i use functional component

**Comment by veraLX** [comment]
Hello，我是林昕，我已收到来信~
